demos/overriding_config/config/terraform.yml

```diff
@@ -0,0 +1,26 @@
+id: "{vars.username}-{vars.project}-{vars.task_name}-{global:cluster}-{global:sut}"
+plan:
+    path: "{mtt_common:terraform.plans.systemtest}"
+state:
+    path: "{_source_:project}/terraform/{id}.terraform.state"
+options:
+    always_init: false
+vars_path: "{_source_:project}/terraform/{id}.tfvars.json"
+vars:
+    manager_count: 1
+    worker_count: 3
+    dtr_count: 0
+    windows_worker_count: 0
+    admin_password: "abcd1234changeme"
+    windows_administrator_password: "tfaws,,CHANGEME..Example"
+    platform: "rhel_7.8"
+    username: "{user:id?user}"
+    task_name: "{global:id}"
+    project: "{global:project}"
+    expire_duration: "72h"
+    kube_orchestration: true
+    dtr_install_flags: ["--ucp-insecure-tls"]
+    platform_repo: "public"
+    engine_version: "19.03.12"
+    ucp_version: "3.3.3"
+    dtr_version: "2.8.3"
```

Comment:
`msr` not `dtr`?

Reply:
these are values that go into the terraform plan tfvars file.  All references are effectively foreign keys that depend on the terraform plan